src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/Pipeline/ExceptionPipeline.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.Diagnostics.Monitoring.StartupHook.Exceptions.Pipeline
+{
+    internal sealed class ExceptionPipeline :
+        IDisposable
+    {
+        private readonly ExceptionPipelineDelegate _exceptionHandler;
```

Comment:
Pipelines:
- Our pipelines
- System.IO.Pipelines
- This.

Maybe a different name?